### ISSUE TITLE
Issue: Accessibility-issues with tags and server listing

### ISSUE THREAD

**Issue by dafluteneko** [BOU4EYT/StoatLists#3]
When only navigating using the keyboard, I was unable at any point to be able to point at the tags or the server listed.

The navigation within keyboard goes something along the lines of:
1. Header (including navbar links)
2. View all
3. Footer links

The expected behavior is:
1. Header (including navbar links)
2. Server tags
3. Server listed + "View all" (as the last choice)
4. Footer links

Those that rely in accessibility features and navigating within the keyboard exclusively simply cannot access any servers that are currently listed, nor can they filter for servers of their choosing.
